### ISSUE TITLE
jlink plugin: need mechanism for explicitly adding additional modules

### ISSUE THREAD

**Issue by barchetta** [oracle/helidon-build-tools#344]
Currently the jlink goal only adds modules based on the project's dependencies:

```
addArgument("--add-modules", String.join(",", javaDependencies));
```
This request is to add the ability to configure the plugin with additional modules. For example a configuration parameter like:

`addModules`: additional modules to pass to jlink's add-modules option

One use case for this is described here:

https://stackoverflow.com/questions/54770538/received-fatal-alert-handshake-failure-in-jlinked-jre

In this case you need to explicitly add the `jdk.crypto.ec` module. One could do this by creating a `module-info.java` file for the project, but one of our goals with the jlink support is not to require this.

**Comment by dansiviter** [comment]
I've had a bash at addressing this. Not sure how to unit test it, but I have tried it manually and looks good.

**Comment by barchetta** [comment]
Need to port to 2.2.0

**Comment by romain-grecourt** [comment]
It was already included in the 2.1.3 release actually.
